pkg/epp/plugins/plugins.go

```diff
@@ -16,9 +16,27 @@ limitations under the License.
 
 package plugins
 
+const (
+	KVCacheUsagePercentKey = "KVCacheUsagePercent"
+	WaitingQueueSizeKey    = "WaitingQueueSize"
+	MaxActiveModelsKey     = "MaxActiveModels"
+	ActiveModelsKey        = "ActiveModels"
+	WaitingModelsKey       = "WaitingModels"
+	UpdateTimeKey          = "UpdateTime"
+)
+
 // Plugin defines the interface for a plugin.
 // This interface should be embedded in all plugins across the code.
 type Plugin interface {
 	// TypedName returns the type and name tuple of this plugin instance.
 	TypedName() TypedName
+	// Consumes represents data consumed by the plugin.
+	Consumes() map[string]any
+}
```

Comment:
what if a plugin doesn't consume any data? 
can we move this function from the general plugin interface to a dedicated interface?
there are multiple examples for plugins that do not consume data, like RandomPicker, ProfilerHandler, and more.